src/middlewares/error-sender.js

```diff
@@ -1,12 +1,12 @@
 import errorsResponses from '../lib/errors-responses';
 
-const errorSender = (error, requset, response, next) => {
+const errorSender = (error, request, response, next) => {
   const errorResponse = errorsResponses[error.name];
   if (errorResponse) {
     response.status(error.status || 500).json(errorResponse);
     next();
   } else {
-    response.status(error.status || 500).json(error.message);
+    response.status(error.status || 500).json({ message: error.message });
```

Comment:
Error should also contain status code and it's name in the object, sent to client

Reply:
Can I fix all like it in PR with errors?

Reply:
Yes